main.tf

```
@@ -42,10 +42,18 @@ locals {
     data_volume_size      = var.cluster_data_volume_size
     server_group_policy   = var.cluster_server_group_policy
     floating_ip           = false
+    k3s_args              = var.cluster_k3s_agent_args
```

```suggestion
    k3s_args              = var.cluster_k3s_args
```

Do I misunderstand something or is this incorrect?

---

Using the `cluster_k3s_agent_args` here is correct, but I forgot to handover `cluster_k3s_args` to the submodule.